src/Schema/Directives/CanDirective.php

```diff
@@ -80,7 +82,16 @@ public function handleField(FieldValue $fieldValue, Closure $next): FieldValue
             $fieldValue->setResolver(
                 function ($root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo) use ($previousResolver) {
                     if ($find = $this->directiveArgValue('find')) {
-                        $modelOrModels = $this->getModelClass()::findOrFail($args[$find]);
+                        $modelOrModels = $resolveInfo
+                            ->argumentSet
+                            ->enhanceBuilder(
+                                $this->getModelClass()::query(),
+                                $this->directiveArgValue('scopes', []),
```

Comment:
Allowing scopes here is a hidden new feature. I don't see a concrete use case for it, so let's remove this option for now.

Reply:
Hm, right